pyanaconda/modules/subscription/subscription_interface.py

```diff
@@ -171,3 +171,12 @@ def RegisterOrganizationKeyWithTask(self) -> ObjPath:
         return TaskContainer.to_object_path(
             self.implementation.register_organization_key_with_task()
         )
+
+    def UnregisterWithTask(self) -> ObjPath:
```

Comment:
This should probably have the @emits_properties_changed decorator.

Reply:
Oh, right, adding it there.

Reply:
Is it really required here? The only change I see is based on signal of the task which means a lot later than when this method ends.

Reply:
@jkonecny12 is right.

Reply:
OK, dropping that decorator. :)